cli/createTable.js

```diff
@@ -0,0 +1,42 @@
+import { table } from 'table';
+
+const tableConfig = {
+    columns: {
+        0: { width: 21, wrapWord: true },
+        1: { width: 33, wrapWord: true },
+        2: { width: 21, wrapWord: true },
+        3: { width: 33, wrapWord: true },
+        4: { width: 10, wrapWord: true },
+        5: { width: 10, wrapWord: true },
+        6: { width: 10, wrapWord: true }
+    }
+};
+
+export function createTable(events, pending) {
```

Comment:
I believe the `pending` filtering logic should be dealt with at the data fetching level.
If we want this module to be consumable by another interface other than the CLI (i.e: an UI), it needs to have that functionality there to not force the consumer to implement it.

The view layer just creates the table from the data:
```suggestion
export function createTable(events) {
```